### PR TITLE
Feat: M2-1447 - Builder - Applet - Activity Items - Connect Single/Multiple Selection Rows to api

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionRows/Items/Items.tsx [hunk 1]
```diff
@@ -26,52 +29,66 @@ const commonUploaderProps = {
...
-      'selectionRows.items',
-      items?.filter((item: SelectionRowsItem, key: number) => key !== index),
+      `${name}.responseValues.rows`,
+      rows?.filter((row: SingleAndMultipleSelectRow, key: number) => key !== index),
     );
 
-  return items?.map((item: SelectionRowsItem, index: number) => {
-    const name = `selectionRows.items[${index}]`;
+    if (hasScores) {
+      const dataMatrix = getValues(`${name}.responseValues.dataMatrix`);
```
Comment (alex-i-e) [inline src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionRows/Items/Items.tsx:53]: Can you move `${name}.responseValues.dataMatrix` in a variable dataMatrixName to reduce code duplication?

Comment (muhurov) [reply src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/SelectionRows/Items/Items.tsx]: done

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.const.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const DEFAULT_EMPTY_SLIDER_ROWS = {
 export const DEFAULT_AUDIO_DURATION_SEC = 300;
 export const DEFAULT_MAX_CHARACTERS = 300;
 export const DEFAULT_EMPTY_SELECTION_ROWS_OPTION = { label: '', tooltip: '', image: '' };
-export const DEFAULT_EMPTY_SELECTION_ROWS_ITEM = { label: '', tooltip: '', image: '', scores: [] };
+// export const DEFAULT_EMPTY_SELECTION_ROWS_ITEM = { label: '', tooltip: '', image: '', scores: [] };
```
Comment (IhorNesterenko99) [inline src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.const.tsx:35]: do you need this commented code?

**Review by IhorNesterenko99 — APPROVED** [review]

File: src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.const.tsx (hunk 2 above)
Comment (muhurov) [reply src/modules/Builder/features/ActivityItems/ItemConfiguration/ItemConfiguration.const.tsx]: removed

**Review by alex-i-e — APPROVED** [review]

**Pull request merged**